### DISCUSSION TITLE
Highlight-groups used

### DISCUSSION THREAD

**Discussion by Davincible** [williamboman/nvim-lsp-installer#383] [Q&A] [answered]
What are the highlight groups used for the info window? The colors in my window with my theme happen to be very low contrast, so I want to manually define the colors in my theme.

I found these, in the code while looking for them, but I can't seem to figure out how to change the text foreground and window background;
https://github.com/williamboman/nvim-lsp-installer/blob/e8530f4059279a0d6b5e5f8891d04ffb1ade29a3/lua/nvim-lsp-installer/ui/status-win/init.lua#L824-L836

Edit:
Ah found out its just the `NormalFloat` group. Other question, is it possible to give the window a border, like the Telescope window.

**Comment by williamboman** [comment, marked as answer]
Hello! Yeah those are the custom highlight groups used in the info window. To be honest, they are just thrown together without much forethought. In some places, standard highlight groups are used (like `Comment` and `String`, `Boolean`, etc.). I would like to have better interop with different colorschemes and leverage standard highlight groups a bit more, but it's not something I'm planning on looking into, at least in the near future.

As for borders - there should be borders already, styled using the default `FloatBorder` hl group

**Reply by williamboman** [reply]
You could override these custom highlight groups in your own config if you'd like to. However, I don't really consider them part of the public API, and so they may change without notice in the future.
